### ISSUE TITLE
How to add rotation augmentation option to data transforms?

### ISSUE THREAD

**Issue by wangg12** [ncullen93/torchsample#15]
```
from torchvision import  transforms
import torchsample

data_transform = transforms.Compose([
          transforms.Scale(256),
          torchsample.transforms.Rotate(30),
          transforms.CenterCrop(224),
          transforms.ToTensor(),
          transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])
      ])
```
Is this right? Where is the proper position and proper way to use it?

**Comment by ncullen93** [comment]
That looks almost right to me. You might need it after 'ToTensor' though since it works on torch.Tensor. The issue with a lot of the actual pytorch transforms is that they only work on PIL images or numpy arrays instead of torch tensors

**Comment by wangg12** [comment]
@ncullen93 So I need to put `Rotate(30)` after `ToTensor`? BTW, the docstring of Rotate about lazy seems wrong. https://github.com/ncullen93/torchsample/blob/master/torchsample/transforms/affine_transforms.py#L184

**Comment by ncullen93** [comment]
correct. I believe all or nearly all of the torchsample transforms operate directly on torch tensors. Therefore, if you're reading images you should call `ToTensor` first before any of them. and correct, they're switched thanks. I'll fix it.

**Comment by wangg12** [comment]
@ncullen93 The `fill_mode` argument is not implemented?

**Comment by ncullen93** [comment]
No it just fills with 0's. Are you looking for something specific? I can implement it pretty quickly, but it will slow things down a fraction.

**Comment by wangg12** [comment]
Is filling 0 a common practice?

**Comment by ncullen93** [comment]
Yes if your data is normalized to 0-1 or some constant otherwise. It's what I use

**Comment by wangg12** [comment]
OK, Thanks.
